mibipret/data/names_data.py

```diff
@@ -74,16 +75,82 @@
 name_indene = 'indene'
 name_naphthalene = 'naphthalene'
 
-### todo: complete and potentially choose other names
+### Standard names for additional contaminants
+name_styrene = 'styrene'
```

Comment:
As discussed, we can have a message encouraging users to open issues (complete with URL to a custom template where possible) with any missing contaminant names etc.

Reply:
Ah I see it is already there in #67